api/database/functions.py

```diff
@@ -151,22 +228,24 @@ async def verify_token(token:str, verification:str, route:str=None) -> bool:
             insert_usage = insert(ApiUsage).values(insert_values)
             await session.execute(insert_usage)
             await session.commit()
-        
+
     # If len api_user == 0; user does not have necessary permissions
     if len(api_user) == 0:
-        raise HTTPException(status_code=401, detail=f"Insufficent Permissions: Either the token does not exist or you don't have sufficent permissions to access this content.")
-    
+        raise HTTPException(
+            status_code=401, detail=f"Insufficent Permissions: Either the token does not exist or you don't have sufficent permissions to access this content.")
+
     api_user = api_user[0]
-    
+
     if api_user['is_active'] != 1:
-        raise HTTPException(status_code=403, detail=f"User token has been disabled. Please contact a developer.\nThis could be due to having an inactive token (>30d) or a manual shutdown by a developer.")
-    
+        raise HTTPException(
+            status_code=403, detail=f"User token has been disabled. Please contact a developer.\nThis could be due to having an inactive token (>30d) or a manual shutdown by a developer.")
```

Comment:
raise HTTPException(status_code=401, detail=f"Insufficent Permissions")

Reply:
403 indicates valid credentials without necessary permissions. 401 indicates invalid credentials - will not be changed.